urls.py

```diff
@@ -6,11 +6,11 @@
     'corehq.apps.prelogin.views',
     url(r'^home/$', HomePublicView.as_view(), name=HomePublicView.urlname),
     url(r'^impact/$', ImpactPublicView.as_view(), name=ImpactPublicView.urlname),
-    url(r'^software_services/$', SoftwareServicesPublicView.as_view(),
+    url(r'^pricing/$', SoftwareServicesPublicView.as_view(),
         name=SoftwareServicesPublicView.urlname),
     url(r'^services/$', ServicesPublicView.as_view(),
         name=ServicesPublicView.urlname),
-    url(r'^pricing/$', PricingPublicView.as_view(),
+    url(r'^services/$', PricingPublicView.as_view(),
```

Comment:
Actually, reading through [FB 219898](http://manage.dimagi.com/default.asp?219898) the language code issue is different. It needs to stay, it needs to cater for `en_us`, and the base template needs to be changed to support it too. So please ignore previous comment.